.vsts/linux-buildbot.yml

```diff
@@ -67,5 +68,5 @@ steps:
 - script: make pythoninfo
   displayName: 'Display build info'
 
-- script: make buildbottest TESTOPTS="-j4 -uall,-cpu"
+- script: xvfb-run make buildbottest TESTOPTS="-j4 -uall,-cpu"
```

Comment:
Didn't you mean to add `gui` to the `-u` argument? (here and in the other files)